modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/TableList.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import classnames from 'classnames';
+import React, {useState} from 'react';
+
+import './index.scss';
+
+export type TableHeaders = {
+	bold?: boolean;
+	key: string;
+	value: string;
+};
+
+type Props = {
+	data: InfoRowContent[];
+	headers: TableHeaders[];
+};
+
+type InfoRowContent = {[keys: string]: string};
+
+const {Body, Cell, Head, Row} = ClayTable;
+
+const TableList: React.FC<Props> = ({data, headers}) => {
```

Comment:
Simplify

1. data is too generic, it could be anything.
2. items is more specific for this purpose

```suggestion
const Table: React.FC<Props> = ({items, headers}) => {
```

Reply:
Done!